docs/migration/Version_0.5.x_0.7.x.md

```diff
@@ -0,0 +1,279 @@
+| `dataplane.token.refresh.expiry_seconds`           | `EDC_DATAPLANE_TOKEN_EXPIRY`                        |          | 30                   | TTL for access tokens                                     |
+| `dataplane.token.refresh.expiry_tolerance_seconds` | `EDC_DATAPLANE_TOKEN_EXPIRY_TOLERANCE`              |          | 10                   | tolerance for token expiry                                |
+| `dataplane.token.refresh.refresh_endpoint`         | `EDC_DATAPLANE_TOKEN_REFRESH_ENDPOINT`              |          | `<PUBLIC_API>/token` | endpoint for an OAuth2 token refresh request              |
+| `dataplane.token.refresh.refresh_endpoint`         | `EDC_DATAPLANE_TOKEN_REFRESH_ENDPOINT`              |          | `<PUBLIC_API>/token` | endpoint for an OAuth2 token refresh request              |
+| `dataplane.token.signer.privatekey_alias`          | `EDC_TRANSFER_PROXY_TOKEN_SIGNER_PRIVATEKEY_ALIAS`  | x        |                      | alias, under which the private key is stored in the vault |
+| `dataplane.token.verifier.publickey_alias`         | `EDC_TRANSFER_PROXY_TOKEN_VERIFIER_PUBLICKEY_ALIAS` | x        |                      | alias, under which the public key is stored in the vault  |
+| `dataplane.endpoints.signaling.port`               | `WEB_HTTP_SIGNALING_PORT`                           |          | 8083                 | port where the Signaling API is exposed                   |
+| `dataplane.endpoints.signaling.path`               | `WEB_HTTP_SIGNALING_PATH`                           |          | `/api/signaling`     | path where the Signaling API is exposed                   |
+
+### 1.6 Further references
+
+- Dataplane Signaling
+  [documentation](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/data-plane-signaling/data-plane-signaling.md)
+  and [token
+  handling](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/data-plane-signaling/data-plane-signaling-token-handling.md)
+- [Tractus-X Specification for Token
+  Renewal](https://github.com/eclipse-tractusx/tractusx-profiles/blob/main/tx/refresh/refresh.token.grant.profile.md)
+- [Tractus-X EDC Implementation
+  Documentation](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/development/dataplane-signaling/tx-signaling.extensions.md)
+
+## 2. Changes to EDR handling and APIs
+
+The setting `backendService.httpProxyTokenReceiverUrl`, which configured an EDR receiver backend using the upstream
+EDC [extension](https://github.com/eclipse-edc/Connector/tree/main/extensions/control-plane/transfer/transfer-pull-http-dynamic-receiver)
+is no longer available as well as the extension is not shipped in the Tractus-X EDC 0.7.x distributions.
+
+The main reason is that EDC/Tractus-X-Edc switched to Dataplane Signaling for handling transfers and the EDR format
```

Comment:
```suggestion
The main reason is that EDC/Tractus-X-EDC switched to Dataplane Signaling for handling transfers and the EDR format
```
Nitpicking here since it irks me to see EDC written differently in the same sentence.